src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -127,7 +128,7 @@ public void registerCivilian(final AbstractCivilianEntity entity)
             return;
         }
 
-        if (!existingCitizen.get().isAlive() || !existingCitizen.get().world.isBlockPresent(existingCitizen.get().getPosition()))
+        if (!existingCitizen.get().isAlive() || !WorldUtil.isBlockLoaded(existingCitizen.get().world, existingCitizen.get().getPosition()))
```

Comment:
think this should be an entity block loaded check, as its for an existing citizen in a possibly not loaded chunk, we want to check if its a chunk which ticks entities